### PR TITLE
kitty: Add shell completion files

### PR DESCRIPTION
**Pull request by judaew**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.2.3 20D91 on x86_64
Xcode 12.4 12D4e

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: aqua/kitty/Portfile [hunk 1]
```diff
@@ -47,7 +47,7 @@ depends_lib-append  port:harfbuzz
 
 build.cmd           "${python.bin} setup.py"
 build.target        kitty.app
-build.args-append   --verbose
+build.args-append   --verbose --update-check-interval=0
```
Comment (herbygillot) [inline aqua/kitty/Portfile:50]: Does this disable kitty checking for updates?

**Comment by herbygillot** [comment]
Also please add `openmaintainer` to the `maintainers` section unless you have a specific reason for doing so.

File: aqua/kitty/Portfile (hunk 1 above)
Comment (judaew) [reply aqua/kitty/Portfile:50]: Yes, it is. 

From [setup.py](https://github.com/kovidgoyal/kitty/blob/cdf64bf01614324200eac8955503e9a3ac432f73/setup.py#L1202-L1207):

> When building a package, the default value for the update_check_interval setting will be set to this number. Use zero to disable update checking.

**Comment by judaew** [comment]
Done

File: aqua/kitty/Portfile (hunk 1 above)
Comment (herbygillot) [reply aqua/kitty/Portfile:50]: OK np

**Pull request merged**